py/qqa/qa/runner.py

```diff
@@ -100,17 +100,20 @@ def run(self, indir, outfile=None):
             PER_EXP = ['NIGHT', 'EXPID'],
         )
 
-        from pkg_resources import resource_filename
-        jsonfiledir = resource_filename('qqa', os.path.join('cal_files', 'timeseries_dropdown.json'))
-        if os.path.isdir(os.path.dirname(jsonfiledir)):
-            with open(jsonfiledir, 'w') as out:
-                result1 = dict()
-                for key1 in results:
-                    colnames_lst = results[key1][0].colnames
-                    for i in join_keys[key1]:
-                        colnames_lst.remove(i)
-                    result1[key1] = colnames_lst
-                json.dump(result1, out)
+        if json_gen:
+            from pkg_resources import resource_filename
+            jsonfiledir = resource_filename('qqa', os.path.join('cal_files', 'timeseries_dropdown.json'))
+            if os.path.isdir(os.path.dirname(jsonfiledir)):
+                with open(jsonfiledir, 'w') as out:
+                    result1 = dict()
+                    for key1 in results:
+                        if key1 != "PER_CAMFIBER":
+                            colnames_lst = results[key1][0].colnames
+                            for i in join_keys[key1]:
+                                colnames_lst.remove(i)
+                            result1[key1] = colnames_lst
+                    json.dump(result1, out)
+                print('Wrote {}'.format(jsonfiledir))
```

Comment:
Since different flavors of exposures can have different QA metrics, timeseries_dropdown.json should have a superset of the QA metrics from all previous exposures, not just the ones that happened to appear in the latest one.  i.e. read in the previous file, add to it, then write out.